marketplace.xml

```diff
@@ -698,6 +698,69 @@
     </screenshots>
   </market_entry>
 
+
+  <market_entry>
+    <id>pentaho-jpivot-plugin</id>
+    <type>Platform</type>
+
+    <name>Pentaho Platform JPivot Plugin</name>
+    <description>
+      This is a port of the JPivot functionality that existed in Pentaho Platform 4.8 into a stand alone plugin.
+    </description>
+
+    <img/>
+    <small_img/>
+
+    <category>
+      <name>Analysis</name>
+      <parent>
+        <name>Apps</name>
+      </parent>
+    </category>
+
+    <license>CPL-1.0</license>
+
+    <installation_notes>
+      This plugin requires a jar (JavaServer Faces API) that was removed form the Pentaho Server.
```

Comment:
I'd suggest something like:
«
To function, this plugin requires that a JAR dependency is installed in the Pentaho Server - the JavaServer Faces API. This dependency has known security vulnerabilities, which are described in the security issue CVE-2008-1285. Installing this dependency in the Pentaho Server compromises its security.

To install the JAR dependency, download it from 'https://mvnrepository.com/artifact/javax.faces/jsf-api/1.1', place it in the 'tomcat/webapps/pentaho/WEB-INF/lib' folder and finally restart the Pentaho Server.
»